docs/Policy/Maintenance/Scenario_Maintenance_Policy.md

```diff
@@ -13,12 +14,14 @@
 - [Skipping Scenarios](#skipping-scenarios)
 
 ## Overview
+
 You'll see in this [RACI Chart](../../Onboarding/RACI_Chart.md) that there is a shared responsibility for the maintenance of these openshift CI scenarios. This document is meant to describe the tasks included in maintaining a scenario and state who should be responsible.
```

Comment:
```suggestion
You'll see in this [RACI Chart](../../Onboarding/RACI_Chart.md) that there is a shared responsibility for the maintenance of these OpenShift CI scenarios. This document is meant to describe the tasks included in maintaining a scenario and state who should be responsible.
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)